CHANGELOG/1.1-CHANGELOG.md

```diff
@@ -2,10 +2,27 @@
 
 ## Table of Contents
 
+- [v1.1.1](#v111)
 - [v1.1.0](#v110)
 - [v1.1.0-rc2](#v110-rc2)
 - [v1.1.0-rc1](#v110-rc1)
 
+# v1.1.1
```

Comment:
Since we've been much better at cherry-picking PRs than actually releasing them, this changelog is unfortunately incomplete. Recommend looking at https://github.com/kubernetes-sigs/gateway-api/compare/v1.1.0...release-1.1 for the full set of changes.

Reply:
Oh yes, I missed those. It should be up to date now :)